test/parallel/test-runner-run.mjs

```diff
@@ -500,3 +500,42 @@ describe('require(\'node:test\').run', { concurrency: true }, () => {
     for await (const _ of stream);
   });
 });
+
+describe('coverage report for test stream', () => {
```

Comment:
You have 3 tests here with the same name, please rename

Reply:
my bad 😞.
fixed now 🚀